### PR TITLE
Use standalone Alertmanager chart and Node Exporter's SA config fix

### PR DESCRIPTION
**Pull request by chrissng**
Switch to use the standalone official alertmanager chart.

Also fix SA name and annotation configuration on the node exporter chart.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -379,45 +299,27 @@ variable "alertmanager_service_type" {
...
-  description = "PodSecurityPolicy annotations for alertmanager"
-  default = {
-    "seccomp.security.alpha.kubernetes.io/allowedProfileNames" = "docker/default,runtime/default"
-    "apparmor.security.beta.kubernetes.io/allowedProfileNames" = "runtime/default"
-    "seccomp.security.alpha.kubernetes.io/defaultProfileName"  = "runtime/default"
-    "apparmor.security.beta.kubernetes.io/defaultProfileName"  = "runtime/default"
-  }
-}
-
-variable "alertmanager_pdb_enable" {
```
Comment (chrissng) [inline variables.tf:392]: Standalone chart has no PDB

Comment (lawliet89) [reply variables.tf:392]: Should we use the K8S Terraform resource to create one?

Comment (chrissng) [reply variables.tf:392]: I thought about it but realized this module manages helm resources only.

Comment (chrissng) [reply variables.tf:392]: I'll probably just add it to that chart.

**Review by lawliet89 — APPROVED** [review]

**Pull request merged**